src/main/resources/db/migration/V2_3__create_table_person.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE PERSON
+(
+    id                                  SERIAL               PRIMARY KEY,
+    uuid                                CHAR(36)             NOT NULL UNIQUE,
+    created_at                          timestamptz          NOT NULL,
+    personident                         CHAR(11)             NOT NULL UNIQUE,
+    dodsdato                            DATE
+);
```

Comment:
Trenger man en ny tabell kun for dødsdato, egentlig? Kan ikke `OPPFOLGINGSTILFELLE_PERSON` brukes? 

Må huske på å oppdatere ident på denne tabellen ved identendring også, forresten 😬 Og når `personident` er UNIQUE så kan det også oppstå problemer ved oppdatering av ident. Sånn går dagan..

Reply:
Ja, synes det er best å ha det i egen tabell siden det kan være mange forekomster av oppfolgingstilfelle_person for en og samme personident.

Tror ikke det er så viktig å oppdatere person-tabellen her så lenge den bare brukes til dette (neppe mange som endrer personident etter at de er døde).

Reply:
> Ja, synes det er best å ha det i egen tabell siden det kan være mange forekomster av oppfolgingstilfelle_person for en og samme personident.

Godt poeng!

> Tror ikke det er så viktig å oppdatere person-tabellen ved identendringer så lenge den bare brukes til dødsfalldato (neppe mange som endrer personident etter at de er døde).

Ja, det er også et godt poeng! Eneste måten noen havner i denne tabellen på er vel om de er døde, så da er det ikke så viktig nei! 🌟